### PR TITLE
CI/Simplify e2e ci processes

### PR DESCRIPTION
**Pull request by iGxnon**
Base on #27 

Please briefly answer these questions:

* what problem are you trying to solve? (or if there's no problem, what's the motivation for this change?)

    Just splitting some complex CI processes to enhance the stability of CI. Although such changes will reduce the chances of CI discovering issues, more stringent and rigorous tests should be executed by specific testing scripts rather than running within CI.

* what changes does this pull request make?

* are there any non-obvious implications of these changes? (does it break compatibility with previous versions, etc)

### CODE REVIEW

File: tests/e2e/cases/manifests/operators.yml [hunk 1]
```diff
@@ -17,5 +17,10 @@ spec:
     spec:
       containers:
         - name: xline-operator
-          image: datenlord/xline-operator:latest
+          image: xline-kv/xline-operator:latest
```
Comment (liangyuanpeng) [inline tests/e2e/cases/manifests/operators.yml:20]: Is it possible that `ghcr.io/xline-kv/xline-operator:latest`? just same with `ghcr.io/xline-kv/xline:latest`

Comment (iGxnon) [reply tests/e2e/cases/manifests/operators.yml:20]: No, the image is built locally before running this e2e script. So the name does not matter.

File: tests/e2e/cases/ci.sh [hunk 2]
```diff
@@ -69,28 +74,22 @@ function test::ci::_scale_cluster() {
   log::info "cluster scaled to $1"
 }
 
-function test::ci::_chaos() {
+function test::ci::_random_crash() {
   size=$1
-  iters=$2
   max_kill=$((size / 2))
```
Comment (bsbds) [inline tests/e2e/cases/ci.sh:79]: The max_kill should be size/2+1

Comment (iGxnon) [reply tests/e2e/cases/ci.sh:79]: That is a majority number? Here is max kill

**Review by bsbds — APPROVED** [review]

**Pull request closed without merging**